app/views/users/show.html.haml

```diff
@@ -90,10 +90,10 @@
         %p= t('.proof_of_membership_describe')
         = link_to(t('.download_image'),
                                    proof_of_membership_path(@user, render_to: :jpg),
-                                   class: 'btn btn-sm btn-secondary')
+                                   class: 'btn btn-sm btn-primary')
         = link_to(t('.show_image'),
```

Comment:
We now need to change the functionality:  The action of this button needs to (1) get the short url for the proof of membership, (2) copy it to the user's clipboard, and then (3) show a message to the user that the information has been copied to their clipboard and that they can now paste it where needed.

Just glancing quickly at the code (you should look more carefully at it to verify), seems that you need to use the `short_proof_of_membership_url(user)` method to get the url.

We'll need new text in the locale files for (3) above.  See my review comments in the `sv.yml` file.

Reply:
We'll need the same stuff for the Company H-markt  (need to copy the URL to the clipboard instead of showing the image and add the entries to the locale files)